hardhat.config.ts

```diff
@@ -32,12 +32,17 @@ if (PK) {
   };
 }
 
+let network = "";
+Promise.resolve(argv).then((x) => {
+  network = x.network;
+});
+
 if (
-  ["rinkeby", "mainnet", "goerli", "ropsten"].includes(argv.network) &&
+  ["rinkeby", "mainnet", "goerli", "ropsten"].includes(network) &&
```

Comment:
`network` will always be `""` here as this runs synchronously while the promise handler is pushed to the end of the task queue.

Reply:
You are correct. Thank you. The latest commit merged an update that removed this.